pkg/agent/config/config.go

```diff
@@ -419,6 +443,7 @@ func get(ctx context.Context, envInfo *cmds.Agent, proxy proxy.Proxy) (*config.N
 		ContainerRuntimeEndpoint: envInfo.ContainerRuntimeEndpoint,
 		FlannelBackend:           controlConfig.FlannelBackend,
 		ServerHTTPSPort:          controlConfig.HTTPSPort,
+		Token:                    info.String(),
```

Comment:
This is not the fault of this PR nor is it necessarily the responsibility of this PR to fix but that String implementation seems unwise, maybe it should be `info.Token()`

Reply:
Yeah, I probably should have changed that when refactoring the token stuff :/